src/App.tsx

```diff
@@ -1,61 +1,34 @@
+import React, { useState } from 'react';
+
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import todosFromServer from './api/todos';
```

Comment:
Checklist item 5: Prepare data in one place (App component) and pass it to child components. Don't use `import` keyword in non-root components. This import should be moved to the App component, and the data should be passed down to the TodoForm component as props.